### ISSUE TITLE
abnormal adipose tissue noradrenaline turnover

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3471]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a. abnormal adipose tissue noradrenaline turnover
b. decreased adipose tissue noradrenaline turnover
c.  increased adipose tissue noradrenaline turnover

>2. Synonyms (e.g., absent spleen)

a. abnormal adipose tissue norepinephrine turnover
b. decreased adipose tissue norepinephrine turnover
c.  increased adipose tissue norepinephrine turnover

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a. aberrant level of turnover of the hormone secreted by the adrenal medulla and neurotransmitter in the sympathetic peripheral nervous system and in some tracts of the CNS
b. reduced level of turnover of the hormone secreted by the adrenal medulla and neurotransmitter in the sympathetic peripheral nervous system and in some tracts of the CNS
c. higher level of turnover of the hormone secreted by the adrenal medulla and neurotransmitter in the sympathetic peripheral nervous system and in some tracts of the CNS

PMID:23555928

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. abnormal noradrenaline level (MP:0003964)
b. abnormal adipose tissue noradrenaline turnover (new term)
c. abnormal adipose tissue noradrenaline turnover (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-9196-0135

**Comment by anna-anagnostop** [comment]
See also  abnormal neurotransmitter turnover [#3349, closed](https://github.com/mgijax/mammalian-phenotype-ontology/issues/3349)

**Comment by sbello** [comment]
kicked neurotransmitter turnover out of nervous system placed under homeostasis

**Comment by sbello** [comment]
We did not have neurotransmitter turnover under neurotransmitter level. should it be or do we not put this term under level?

**Comment by sbello** [comment]
MP:0021021 - abnormal 
MP:0021022 - decreased
MP:0021023 - increased
child of abnormal metabolism and abnormal adipose tissue physiology
